python/tests/test_ret.py

```diff
@@ -0,0 +1,25 @@
+from ecdsa import Ed25519
+
+from rola.models.proof import Proof
+from rola.models.signed_challenge import SignedChallenge
+from rola.utils.ret import derive_address
+
+def test_derive_address_from_proof_with_type_person():
```

Comment:
are you testing persona and account proof verification?

Reply:
nope. just persona. can certainly add account too